docs/developing/integrate/middleware/the-graph.md

```diff
@@ -0,0 +1,177 @@
+---
+title: The Graph
+proofedDate: na
+iterationBy: na
+includedInSite: true
+approvedBy: na
+comment: original item https://medium.com/neon-labs/the-graph-on-neon-evm-enabling-efficient-on-chain-dapp-data-querying-d5c73e3c6bb1 the examples repo has (out of date) full proxy local/Solana/Tracer/Indexer for Graph here https://github.com/neonlabsorg/examples/blob/main/the-graph-integration/README.md For the graph we will have separate RPC Endpoint not public 
+---
+
+*This page presents a high-level overview of how to create a subgraph for The Graph and the endpoint services provided by Neon EVM.*
+
+## TL;DR
+
+- Work from The Graph CLI
+
+> `yarn global add @graphprotocol/graph-cli` or `npm install -g @graphprotocol/graph-cli`
+
+- Endpoints:
+    - IPFS: https://ipfs.neonevm.org
+    - Graph UI: https://thegraph.neonevm.org
+    - Deployment: https://thegraph.neonevm.org/deploy
+    - GraphQL: https://thegraph.neonevm.org/index-node/graphql
+
+## Prerequisites for standard flow
+
+:::info 
+
+The standard flow is where the end user deploys and collects data from their own contract. In the standard flow you: 
+
+1. Deploy a contract to Neon EVM
+2. Collect the contract's address and block number
+3. [Configure your subgraph.yaml](#subgraph-overview) to collect data on events emitted by contract with value and block number from previous step
+4. [Deploy your subgraph](#deploy-your-subgraph)
+
+ The Graph is highly customizable and supports other flows, for example it allows you to monitor [public contracts](#public-contracts) too. 
+
+::: 
+
+- Smart contract address on Neon EVM 
+- ABI file for the deployed contract 
+- Graph CLI: see [The Graph docs](https://thegraph.com/docs/en/cookbook/quick-start)
+
+## Neon EVM-specific deviations
```

Comment:
@BoogVAr -- I didn't say much, can you validate that what I did say is correct pls?